cache-wranglerjs-binary/index.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * This Worker is responsible for caching wranglerjs binaries.
+ */
+
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event.request))
+})
+
+/**
+ * Fetch and log a request
+ * @param {Request} request
+ */
+async function handleRequest(request) {
+  let urlParts = /^https?:\/\/workers\.cloudflare\.com\/get\-wranglerjs\-binary\/([^\/]+)\/([^\/]+)\.tar\.gz/.exec(request.url)
+  if (!urlParts) {
+    return new Response("Missing URL components", {status: 400})
```

Comment:
it seems like this is the same across every worker - is there a reason these are all separate workers scripts and not a single worker with routing handled internally?